### PR TITLE
Complete Project

### PR DESCRIPTION
**Pull request by bell2041**

### CODE REVIEW

File: app/Jobs/getAirspaceData.php [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        curl_setopt($process, CURLOPT_POSTFIELDS, json_encode($data_string));
+
+        curl_setopt($process, CURLOPT_SSL_VERIFYHOST, false);
+        curl_setopt($process, CURLOPT_SSL_VERIFYPEER, false);
+
+        $return     = curl_exec($process);
+        $response = json_decode($return, true);
+
+
+        if (Arr::has($response,  "data.color.name")){
```
Comment (joshuaziering) [inline app/Jobs/getAirspaceData.php:73]: nice

File: app/Http/Controllers/FlightsController.php [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+     * @return \Illuminate\Http\RedirectResponse|\Illuminate\Routing\Redirector
+     */
+    public function store(Request $request)
+    {
+
+        $request->validate([
+            'flight_time' => 'required|date|after:'.date('m/d/y h:ia'),
+            'duration_in_seconds' => 'required|int',
+            'notes' => 'required|string',
+            'lat' => ['required','regex:/^[-]?(([0-8]?[0-9])\.(\d+))|(90(\.0+)?)$/'],
```
Comment (joshuaziering) [inline app/Http/Controllers/FlightsController.php:63]: Regex game.

File: app/Http/Controllers/FlightsController.php [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+        $data->delete();
+
+        return redirect('/')->with('success', 'Record has been deleted Successfully');
+    }
+
+
+    /**
+     * @return \Illuminate\Http\JsonResponse
+     */
+    public function apiGetListAll(){
```
Comment (joshuaziering) [inline app/Http/Controllers/FlightsController.php:155]: interesitng

File: app/Jobs/getWeather.php [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+     * @return void
+     */
+    public function handle()
+    {
+        $this->getWeatherByID($this->flightID);
+    }
+
+    public function getWeatherByID($flightID){
+
+        $data = FlightsModel::find($flightID);
```
Comment (joshuaziering) [inline app/Jobs/getWeather.php:40]: Rmb

File: config/app.php [hunk 5]
```diff
@@ -67,7 +67,7 @@
     |
     */
 
-    'timezone' => 'UTC',
+    'timezone' => 'America/New_York',
```
Comment (joshuaziering) [inline config/app.php:70]: Time zones

File: routes/api.php [hunk 6]
```diff
@@ -17,3 +17,7 @@
 Route::middleware('auth:api')->get('/user', function (Request $request) {
     return $request->user();
 });
+
```
Comment (joshuaziering) [inline routes/api.php:20]: Routes

**Comment by bell2041** [comment]
Hi @joshuaziering. 
Would you please take couple of minutes and tell me what needs improvements in my code so I can work on it. It will eat me alive now :) Thank you so much in advance

**Pull request closed without merging**